### ISSUE TITLE
Toast size using position toast-center

### ISSUE THREAD

**Issue by bucari** [robsontenorio/mary#380]
### maryUI version

1.26.2

### daisyUI version

4.10.1

### Livewire version

3.4.10

### What browsers are affected?

Firefox, Chrome, Safari, Microsoft Edge

### What happened?

Hi @robsontenorio 
I noticed that using position: 'toast-center' shrinks width like in the screenshot, whatever the text of the message 

![image](https://github.com/robsontenorio/mary/assets/97893337/9ba19688-b97f-49e5-a5ae-d3a0ae24d32b)

**Comment by robsontenorio** [comment]
Unfortunately  this is a limitation from daisyUI for `toast-center`.

If you try this raw example  from daisyUI you see the issue.
https://daisyui.com/components/toast/#toast-top-toast-center


```html
    <div class="toast toast-top toast-center z-50" >
      <div class="alert alert-info">
          <span  class="font-bold">This is a very very large message for testing something. And long and long ... very very long</span>
      </div>
    </div>
```

As a workaround use `toast-end` or `toast-start`.

**Comment by bucari** [comment]
@robsontenorio that screenshot is not from mobile but desktop

**Comment by robsontenorio** [comment]
Got it. Will be solved by # #385
